DevProxy.Abstractions/Plugins/BasePlugin.cs

```diff
@@ -99,6 +99,15 @@ public TConfiguration Configuration
     }
     public IConfigurationSection ConfigurationSection { get; } = pluginConfigurationSection;
 
+    protected virtual IEnumerable<string>? GetConfigurationValue(string key, IEnumerable<string>? list, IEnumerable<string>? defaultList = default)

```

Comment:
Let's rename `list` to `configuredList` for clarity

Reply:
fixed